### ISSUE TITLE
11 Jul 2024 - A11y Theme Builder Meeting Minutes

### ISSUE THREAD

**Issue by aaronreed708** [finos/a11y-theme-builder#959]
## Date
2024-07-11 - 09:15 EDT

## Zoom info
- Meeting link: https://zoom.us/j/91013899295?pwd=Qnh5UVN1aUZ4ak9tODgvMVNLNUg5Zz09
- Meeting id: 910 1389 9295
- Password: 123225
- Call-in: +1 929 205 6099 US (New York) (Check calendar for more options)

## Untracked attendees
- Fullname, Affiliation, (optional) GitHub username
- ...

## Meeting notices
- FINOS **Project leads** are responsible for observing the FINOS guidelines for [running project meetings](https://community.finos.org/docs/governance/meeting-procedures/). Project maintainers can find additional resources in the [FINOS Maintainers Cheatsheet](https://community.finos.org/docs/finos-maintainers-cheatsheet).

- **All participants** in FINOS project meetings are subject to the [LF Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) and all other [FINOS policies](https://community.finos.org/docs/governance/#policies). 

- FINOS meetings involve participation by industry competitors, and it is the intention of FINOS and the Linux Foundation to conduct all of its activities in accordance with applicable antitrust and competition laws. It is therefore extremely important that attendees adhere to meeting agendas, and be aware of, and not participate in, any activities that are prohibited under applicable US state, federal or foreign antitrust and competition laws. Please contact legal@finos.org with any questions.

- FINOS project meetings may be recorded for use solely by the FINOS team for administration purposes. In very limited instances, and with explicit approval, recordings may be made more widely available.

## Agenda
- [x] Convene & roll call (5mins)
- [x] Display [FINOS Antitrust Policy summary slide](https://community.finos.org/Compliance-Slides/Antitrust-Compliance-Slide.pdf) 
- [x] Review Meeting Notices (see above)
- [x] Approve past meeting minutes
- [x] Apologies for last week miscommunication re: cancelling meeting
- [x] Community WG Update (@PaulaPaul)
    - [x] Update from FINOS OSFF London (@codyzu)
- [x] Pre-commit Hook discussion (@aaronreed708)
- [x] ...
- [x] AOB, Q&A & Adjourn (5mins)

## Minutes
- Paula provided Community WG update
    - Processing badges for June
    - Preparing for Squiggle Conference
    - Grace Hopper Open Source Day confirmed
        - we will have an Accessibility Breakout Room under the FINOS umbrella
        - Lise accepted as speaker
        - Evan, Aaron, Yessel and Cody to participate as able
- Cody provided update on FINOS OSFF London
    - Contacts made at the booth.  Cody to follow up with individuals.
    - Lise presented remotely, it went well
    - Good support from FINOS, very glad we had a presence
- Paula informed us that Karl Moll from FINOS looking to do an integration of multiple projects in preparation for FINOS OSFF NYC
- Aaron  opened a discussion on pre-commit hooks
    - there was no dissenting opinion on phone
    - Aaron did an implementation with a script, but requires develop interaction on local environment to make it work
    - It was suggested to use Husky, which can be part of the NPM install and build
        - It was noted that some developers might not like having this extra control forced on them
    - It was decided that pre-commit hook effort will be done under a separate issue
- Lise updated us on the progress being made on the Theme Builder backend issues
    - Lise, Evan and Keith doing coding sessions to spread the knowledge of the backend code more
- Lise discussed WCAG AA and AAA, raised the question of how to best display them
    - side by side, one or the other
    - currently a toggle

## Decisions Made
- [x] We will address pre-commit hook in a separate issue
- [ ] ...

## Action Items
- [ ] Lise to get Omesh and Aaron CSS and responsive breakpoints and behaviors for #820 design
    - will provide CSS for framework only for now
- [ ] (from last week)Aaron and Evan need to break the Epic that will be worked on by the mentee into detailed and digestable issues
- [ ] (from last week) Lise to work with Aaron to update video in mkdocs documentation
- [x] (from last week) Open issue to evaluate PenPot from Theme Builder perspective
- [ ] Omesh and Aaron need dev mode access from @lwnoble for Figma wireframes in https://github.com/finos/a11y-theme-builder/issues/820
- [x] Aaron to open separate issue for pre-commit hook
- [ ] ...

**Comment by PaulaPaul** [comment]
Paula Paul - Greyshore @PaulaPaul

**Comment by codyzu** [comment]
Cody Zuschlag - Nearform

**Comment by PaulaPaul** [comment]
PenPot Issue (may need to be expanded/refined) can be found here: https://github.com/finos/a11y-theme-builder/issues/665

**Comment by aaronreed708** [comment]
Aaron Reed - DFS
Lise Noble - DFS
Kathleen Burkhardt - BlackRock
Omesh Kumawat

**Comment by aaronreed708** [comment]
Approved in this week's call
